src/Compilers/CSharp/Portable/FlowAnalysis/NullableWalker.cs

```diff
@@ -1076,8 +1075,7 @@ private void ReportSafetyDiagnostic(ErrorCode errorCode, SyntaxNode syntaxNode,
         private void ReportSafetyDiagnostic(ErrorCode errorCode, Location location, params object[] arguments)
```

Comment:
>ReportSafetyDiagnostic [](start = 21, length = 22)

Please merge `ReportNonSafetyDiagnostic`, `ReportSafetyDiagnostic`, `ReportDiagnostic`.

Reply:
I merged `ReportSafetyDiagnostic`, but kept `ReportNonSafetyDiagnostic` as I think it is useful to have a chokepoint for W-warning since it can be reported in multiple places.

---
In reply to: [284913123](https://github.com/dotnet/roslyn/pull/35557#discussion_r284913123) [](ancestors = 284913123)